glide-core/src/protobuf/redis_request.proto

```diff
@@ -38,118 +38,553 @@ message Routes {
+    LMPop                          = 810;
+    LPop                           = 811;
+    LPos                           = 812;
+    LPush                          = 813;
+    LPushX                         = 814;
+    LRange                         = 815;
+    LRem                           = 816;
+    LSet                           = 817;
+    LTrim                          = 818;
+    RPop                           = 819;
+    RPopLPush                      = 820; // deprecated in 6.2.0
+    RPush                          = 821;
+    RPushX                         = 822;
+
+    //// Pub/Sub commands: https://redis.io/docs/latest/commands/?group=pubsub
+
+    PSubscribe                     = 901;
+    Publish                        = 902;
+    PubSubChannels                 = 903;
+    PubSubNumPat                   = 904;
+    PubSubNumSub                   = 905;
+    PubSubShardChannels            = 906;
+    PubSubShardNumSub              = 907;
+    PUnsubscribe                   = 908;
+    SPublish                       = 909;
+    SSubscribe                     = 910;
+    Subscribe                      = 911;
+    SUnsubscribe                   = 912;
+    Unsubscribe                    = 913;
+
+    //// Scripting and Functions commands: https://redis.io/docs/latest/commands/?group=scripting
+
+    Eval                           = 1001;
+    EvalRo                         = 1002;
+    EvalSha                        = 1003;
+    EvalShaRo                      = 1004;
+    FCall                          = 1005;
+    FCallRo                        = 1006;
+    FunctionDelete                 = 1007;
+    FunctionDump                   = 1008;
+    FunctionFlush                  = 1009;
+    FunctionKill                   = 1010;
+    FunctionList                   = 1011;
+    FunctionLoad                   = 1012;
+    FunctionRestore                = 1013;
+    FunctionStats                  = 1014;
+    ScriptDebug                    = 1015;
+    ScriptExists                   = 1016;
+    ScriptFlush                    = 1017;
+    ScriptKill                     = 1018;
+    ScriptLoad                     = 1019;
+
+    //// Server management commands: https://redis.io/docs/latest/commands/?group=server
+
+    AclCat                         = 1101;
+    AclDelUser                     = 1102;
+    AclDryRun                      = 1103;
+    AclGenPass                     = 1104;
+    AclGetUser                     = 1105;
+    AclList                        = 1106;
+    AclLoad                        = 1107;
+    AclLog                         = 1108;
+    AclSave                        = 1109;
+    AclSetSser                     = 1110;
+    AclUsers                       = 1111;
+    AclWhoami                      = 1112;
+    BgRewriteAof                   = 1113;
+    BgSave                         = 1114;
+    Command_                       = 1115; // TODO renaming to `Command` causes protobuf error
```

Comment:
If I rename this to `Command`, protobuf returns
```
  protobuf/redis_request.proto:268:5: "Command" is already defined in "redis_request".
  protobuf/redis_request.proto:268:5: Note that enum values use C++ scoping rules, meaning that enum values are siblings of their type, not children of it.  Therefore, "Command" must be unique within "redis_request", not just within "RequestType".
```
It conflicts with `Command` on line 590. Which one to rename?

Reply:
It would probably impact less code to rename the Command here instead of the one on line 590. However, I think the "right" way to do this is to rename the one at line 590 in a separate PR.